store/shared.go

```diff
@@ -0,0 +1,40 @@
+package store
+
+import (
+	"os"
+
+	"github.com/aws/aws-sdk-go/aws"
+	"github.com/aws/aws-sdk-go/aws/ec2metadata"
+	"github.com/aws/aws-sdk-go/aws/session"
+)
+
+const (
+	RegionEnvVar = "CHAMBER_AWS_REGION"
+)
+
+func getSession(numRetries int) (*session.Session, *string) {
```

Comment:
Looks like `numRetries` is unused, and this is a new function so we could probably lose it.

Reply:
ah, I missed a commit (67ef9eb).  I added the retries to session creation to address https://github.com/segmentio/chamber/issues/92